src/altinn-app-frontend/src/components/base/RadioButtonsContainerComponent.tsx

```diff
@@ -26,6 +25,7 @@ export interface IRadioButtonsContainerProps extends IComponentProps {
   mapping?: IMapping;
   layout?: LayoutStyle;
   source?: IOptionSource;
+  likertDisplay?: 'desktop' | 'mobile';
```

Comment:
Desktop/mobile view is decided in The Container component (RepeatingGroupsLikertContainer.tsx). It could be decided in the Likert component, but it would have to be in both places. I did it in the Container component because the desktop view requires a table layout, where the likert component is a table row. The row is dependent on that there exists a table layout containing it. In my opinion this is best to control on the container.

Reply:
Hmm I think that some old comments I added have made their way in here, sorry about that. The comments on code that is marked as "outdated" can be ignored. See the comment below on using `layout` prop instead, and f.eks. adding a `table` value to the LayoutStyle enum. Since mobile view is basically just the same as a regular radiobuttons group, we really only need 1 setting?